src/software/RigCalibration/main_rigCalibration.cpp

```diff
@@ -194,17 +207,14 @@ int main(int argc, char** argv)
           "The sfm_data.json kind of file generated by OpenMVG.")
       ("descriptorPath", po::value<std::string>(&descriptorsFolder)->required(),
           "Folder containing the .desc.")
-      ("mediapath", po::value<std::string>(&mediaPath)->required(),
-          "The folder path containing all the synchronised image subfolders "
-          "assocated to each camera")
-      ("filelist", po::value<std::string>(&filelist),
-          "An optional txt file containing the images to use for calibration. "
-          "This file must have the same name in each camera folder and contains "
-          "the list of images to load.")
+      ("mediapath", po::value<std::vector<std::string> >(&mediaPath)->multitoken()->required(),
+          "The path(s) containing, for each camera of the rig, the video, "
```

Comment:
"The path to the video file, the folder of the image sequence or a text file (one image path per line) for each camera of the rig (eg. --mediapath /path/to/cam1.mov /path/to/cam2.mov)."